data-integration/utilities.py

```diff
@@ -165,18 +163,18 @@ def datasetExists(
     ds_id : int.
     dbURL : str, optional
         The default is "postgresql://test:example@localhost:5433/dataset".
-    tables : list of strings, optional
-        The default is ["datasets","spatial","data"].
     Returns
     -------
     bool
     """
+    tables = ["datasets", "spatial", "data"]
     engine = sqla.create_engine(dbURL)
     lengths = []
     for table in tables:
         with engine.connect() as con:
             rs = con.execute(
-                "SELECT COUNT(*) FROM {} WHERE ds_id = {}".format(table, ds_id)
+                "SELECT COUNT(*) FROM {} WHERE ds_id = %(ds_id)s;".format(table),
+                {"ds_id": ds_id},
             )
             for row in rs:
                 count = row[0]
```

Comment:
Then the count variable is reassigned on each iteration and only the last value is used ... is it what you want??